packages/directory/tests/integration/components/dir-sidebar-test.js

```diff
@@ -0,0 +1,58 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('dir-sidebar', 'Integration | Component | dir sidebar', {
```

Comment:
We should start moving to the new testing format as shown here
https://github.com/emberjs/ember-qunit/blob/master/docs/migration.md

Reply:
We are still on 2.18, i think these changes come with 3.x.
I ran the code-mods, no change